pandas/io/formats/html.py

```diff
@@ -74,9 +77,19 @@ def _write_cell(self, s, kind='td', indent=0, tags=None):
                                ('>', r'&gt;')])
         else:
             esc = {}
+
         rs = pprint_thing(s, escape_chars=esc).strip()
-        self.write(u'{start}{rs}</{kind}>'
-                   .format(start=start_tag, rs=rs, kind=kind), indent)
+
+        if self.urls_as_links and _is_url(rs):
+            rs_unescaped = pprint_thing(s, escape_chars={}).strip()
+            start_tag += '<a href="{url}">'.format(url=rs_unescaped)
+            end_a = '</a>'
+        else:
+            end_a = ''
+
+        self.write(u'{start}{rs}{end_a}</{kind}>'
```

Comment:
Hmm is unicode required here? We drop Py27 support soon

Reply:
It seems current Travis and Azure failures occur when the cell to write is a Greek sigma:

`UnicodeEncodeError: 'ascii' codec can't encode character u'\u03c3' in position 0: ordinal not in range(128)`

I don't know much about this, is there an alternative to using the unicode string?

Reply:
@benarthur91 : Revert the change here and write unicode if that's the case.